src/app/modules/instructor/interfacing-instructor.service.ts

```diff
@@ -101,8 +102,7 @@ export class InterfacingInstructor {
   }
```

Comment:
Do we need to modify passCongratulations above? When modifying a function (e.g., like you did `presentMessage`) ensure you check all calls of that function.